docs/hdfs_connector.rst

```diff
@@ -54,16 +82,37 @@ The three records entered are published to the Kafka topic ``test_hdfs`` in Avro
 
 Before starting the connector, please make sure that the configurations in
 ``etc/kafka-connect-hdfs/quickstart-hdfs.properties`` are properly set to your configurations of
-Hadoop, e.g. ``hdfs.url`` points to the proper HDFS and using FQDN in the host. Then run the
-following command to start Kafka connect with the HDFS connector::
+Hadoop, e.g. ``hdfs.url`` points to the proper HDFS and using FQDN in the host. Then start connector by loading its
+configuration with the following command:
+
+.. sourcecode:: bash
+
+   $ confluent load hdfs-sink -d etc/kafka-connect-hdfs/quickstart-hdfs.properties
+   {
+     "name": "hdfs-sink",
+     "config": {
+       "connector.class": "io.confluent.connect.hdfs.HdfsSinkConnector",
+       "tasks.max": "1",
+       "topics": "test_hdfs",
+       "hdfs.url": "hdfs://localhost:9000",
+       "flush.size": "3",
+       "name": "hdfs-sink"
+     },
+     "tasks": []
+   }
+
+To check that the connector started successfully open the Connect worker's log by running:
+to HDFS.
 
+.. sourcecode:: bash
 
-  $ ./bin/connect-standalone etc/schema-registry/connect-avro-standalone.properties \
-  etc/kafka-connect-hdfs/quickstart-hdfs.properties
+  $ confluent log connect
 
-You should see that the process starts up and logs some messages, and then exports data from Kafka
-to HDFS. Once the connector finishes ingesting data to HDFS, check that the data is available
-in HDFS::
+By navigating at the end of the log, you should see that the connector starts, logs a few messages, and then exports
```

Comment:
Using "navigating" sounds like they have to find the log file. Perhaps "The last lines in the log should show that the connector starts ...".